src/AltinnCore/Designer/gulpfile.js

```diff
@@ -118,23 +129,20 @@ function deleteReactCss() {
 }
 
 function setupWatchers(cb) {
-  const jsFile = '../../react-apps/ux-editor/dist/react-app.js';
-  const cssFile = '../../react-apps/ux-editor/dist/react-app.css';
-
   var checkJsFile = setInterval(function () {
-    if (fs.existsSync(jsFile)) {
-      jsWatcher = chokidar.watch(jsFile);
-      jsWatcher.on('ready', copyReactJs);
+    if (fs.existsSync(jsOrgFile)) {
+      jsWatcher = chokidar.watch(jsOrgFile);
+      // jsWatcher.on('ready', copyReactJs);
```

Comment:
Bør dette bare slettes?

Reply:
From @clarkt initial comment:
`jswatcher on.ready commented out, in case we find we need to add it back in.`